movingpandas/trajectory_generalizer.py

```diff
@@ -200,3 +201,50 @@ def _generalize_traj(self, traj, tolerance):
         new_df = traj.df.iloc[keep_rows]
         new_traj = Trajectory(new_df, traj.id)
         return new_traj
+
+
+class TopDownTimeRatioGeneralizer(TrajectoryGeneralizer):
+    """
+    Generalizes using Top-Down Time Ratio algorithm proposed by Meratnia and De By (DOI: 10.1007/978-3-540-24741-8_44).
```

Comment:
Please include a short description of the overall idea of the algorithm so users don't need to go back to the original paper to understand the basics.

Reply:
Ok!